### ISSUE TITLE
what is the purpose of the |updatedisclaimer| ?

### ISSUE THREAD

**Issue by DelazJ** [qgis/QGIS-Documentation#1382]
According to rst_prolog tag defined in https://github.com/qgis/QGIS-Documentation/blob/master/source/conf.py#L219, the |updatedisclaimer| reference is supposed to be at the top of each page of testing doc and point people to the released doc.
Actually, not all pages in testing doc have this mention (e.g. http://docs.qgis.org/testing/en/docs/user_manual/working_with_vector/attribute_table.html). There are also [some files](https://github.com/qgis/QGIS-Documentation/edit/master/source/docs/user_manual/plugins/plugins_coordinate_capture.rst) with the following instruction: `.. comment out this Section (by putting '|updatedisclaimer|' on top) if file is not up to date with release`  leading to another usage of this |updatedisclaimer| (and subsequently, its removal from  some files).
So, anyone knows:
- what is the real purpose of this tag?
- how it can be updated/cleanly removed (see #1252 or #1381) once the doc is released?

**Comment by yjacolin** [comment]
`updatedisclaimer` tag adds a sentence only for testing manual to alert reader that this is a testing documentation and not the official documentation.

For the last item, I need to do some test as your tests were not successfull.

**Comment by DelazJ** [comment]
@yjacolin 
> updatedisclaimer tag adds a sentence only for testing manual to alert reader that this is a testing documentation and not the official documentation.

Yes this is the meaning I opted for and that leads me in #1391 to remove the comment about file update status.

What actually puzzles me is "how is [rst_prolog](https://github.com/qgis/QGIS-Documentation/blob/master/source/conf.py#L219) tag is supposed to work"? Sphinx doc [defines] (http://www.sphinx-doc.org/en/stable/config.html#confval-rst_prolog) it as: `A string of reStructuredText that will be included at the beginning of every source file that is read.`
1/ Doesn't it mean that once defined in the `conf.py` file it's automatically added to the top of each file when generating the html (only this format?)? I failed to have it worked like that but maybe I missed something.
2/ If we have to manually add |updatedisclaimer| to each file, what's the rationale to define it as a rst_prolog tag? Defining it as a simple substitution (like done with |qg|...) would have done the trick, isn't it?
